src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -482,7 +441,9 @@ public function isSubclassOf($class)
             return strtolower($parentClassName);
         }, $realParentClassNames), $realParentClassNames);
 
-        $realParentClassName = $parentClassNames[strtolower($class)] ?? $class;
+        $lowercasedClass = strtolower($class);
```

Comment:
Was the type checker choking here, due to a missing variable? 🤔

Reply:
`PossiblyInvalidArrayOffset - src/Reflection/Adapter/ReflectionClass.php:444:32 - Cannot access value on variable $parentClassNames using a lowercase-string offset, expecting non-empty-lowercase-string (see https://psalm.dev/116)`